src/gConsent/manage/revokeAccessGrant.ts

```diff
@@ -48,19 +49,18 @@ const { quad, namedNode } = DataFactory;
 async function revokeAccessGrant(
   vc: DatasetWithId | VerifiableCredential | URL | UrlString,
   options: Omit<AccessBaseOptions, "accessEndpoint"> = {},
+  type: NamedNode<string> = solidVc.SolidAccessGrant,
```

Comment:
That looks like a good direction. Ideally however, I'd prefer this not to be part of the public API. Could you create a function `revokeAccessCredential` (it can be side-by-side with the current `revokeAccessGrant`) that would have the additional parameter you just added in its signature, and then `revokeAccessGrant` and `cancelAccessRequest` would both call `revokeAccessCredential` with the appropriate parameters.

`revokeAccessCredential` would not be exported by index files that are documented in `package.json`, making it not public API. You can even add the `@internal` TSDoc annotation to make sure the function isn't documented in API docs.